config/locales/en.yml

```diff
@@ -362,11 +362,11 @@ en:
             href: "https://womensaid.scot/"
           - text: "Get information for children from Childline"
             href: "https://www.childline.org.uk/"
-          - text: "Get information if you are child in Wales"
+          - text: "Get information if you‘re child in Wales"
```

Comment:
Should that either be:
- Get information if you‘re a child in Wales, or
- Get information if you‘re child is in Wales